dates.py

```diff
@@ -6,13 +6,13 @@
 import sys
 
 # SEM_BEGIN=datetime.datetime.now(pytz.timezone('Asia/Kolkata'))
-SEM_BEGIN=build_event.generateIndiaTime(2018, 7, 17, 0, 0)
+SEM_BEGIN = build_event.generateIndiaTime(2019, 1, 1, 0, 0)
```

Comment:
This should be 2, I think.

Reply:
This is a bug with python. This happens for all the starting dates. If we put two, the events start from 3 etc.